project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1366 @@
+      - &secondary_heating_type_fuel_fuel_oil
+        option: HVAC Secondary Heating Fuel|Fuel Oil
+        apply_logic:
+          - or:
+            - HVAC Heating Type And Fuel|Fuel Oil Fuel Boiler
+            - HVAC Heating Type And Fuel|Fuel Oil Fuel Wall/Floor Furnace
+      - &secondary_heating_type_fuel_other_fuel
+        option: HVAC Secondary Heating Fuel|Other Fuel
+        apply_logic:
+          - or:
+            - HVAC Heating Type And Fuel|Other Fuel Fuel Boiler
+            - HVAC Heating Type And Fuel|Other Fuel Fuel Wall/Floor Furnace
+      - &secondary_heating_type_fuel_electricity
+        option: HVAC Secondary Heating Fuel|Electricity
+        apply_logic:
+          - or:
+            - HVAC Heating Type And Fuel|Electricity Baseboard
+            - HVAC Heating Type And Fuel|Electricity Electric Boiler
+            - HVAC Heating Type And Fuel|Electricity Electric Wall Furnace
+
+      # Set secondary fuel boiler efficiency levels
+      - &secondary_boiler_76
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 76% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 76% AFUE
+      - &secondary_boiler_80
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 80% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 80% AFUE
+      - &secondary_boiler_90
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 90% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 90% AFUE
+      - &secondary_ele_boiler
+        option: HVAC Secondary Heating Efficiency|Electric Boiler, 100% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Electric Boiler, 100% AFUE
+
+      # Set secondary fuel Wall/Floor Furnace efficiency levels
+      - &secondary_wallfloor_furnace_60
+        option: HVAC Secondary Heating Efficiency|Fuel Wall/Floor Furnace, 60% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 60% AFUE
+      - &secondary_wallfloor_furnace_68
+        option: HVAC Secondary Heating Efficiency|Fuel Wall/Floor Furnace, 68% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 68% AFUE
+      - &secondary_ele_wallfloor_furnace
+        option: HVAC Secondary Heating Efficiency|Electric Wall Furnace, 100% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Electric Wall Furnace, 100% AFUE
+      
+      # Set secondary electric Wall/Floor Furnace efficiency levels
+      - &secondary_ele_baseboard
+        option: HVAC Secondary Heating Efficiency|Electric Baseboard, 100% Efficiency
+        apply_logic:
+          - HVAC Heating Efficiency|Electric Baseboard, 100% Efficiency
+
+      ## ASHP ducted heating ##
+      # Natural Gas as backup
+      - &fed_min_ashp_exist_backup_shared_ducts_60_ng
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 15.05, 8.82 HSPF, Integrated Backup, 60% AFUE NG, 5F-40F switchover band
```

Comment:
we want 0F-40F switchover for fed min ASHP

Reply:
revised